exercises/practice/food-chain/.meta/example.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+declare(strict_types=1);
+
+class FoodChain
+{
+    private static array $verse = [
```

Comment:
You can turn that into a class constant:

```suggestion
    private const VERSES = [
```